libs/model/src/models/community_contract_template.ts

```diff
@@ -24,8 +24,16 @@ export default (
       'CommunityContractTemplate',
       {
         id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true },
-        community_contract_id: { type: dataTypes.INTEGER, allowNull: false },
-        cctmd_id: { type: dataTypes.INTEGER, allowNull: false },
+        community_contract_id: {
+          type: dataTypes.INTEGER,
```

Comment:
It's insane that these columns are auto-incremented in the first place. These are foreign keys to other tables and should not be auto incremented _at all_. We need to create a separate issue for fixing this or add this fix in a migration in this PR. Associations for this model should be reflected in the DB and primary key removed from all columns except `id`.

Reply:
let's create another PR for this please